src/cordova/android/OutlineAndroidLib/outline/src/main/aidl/org/outline/TunnelConfig.aidl

```diff
@@ -14,10 +14,7 @@
 
 package org.outline;
 
-import org.outline.shadowsocks.ShadowsocksConfig;
-
 parcelable TunnelConfig {
   String id;
-  String name;
-  ShadowsocksConfig proxy;
+  String proxyString;  
```

Comment:
The concepts of a service configuration in the app is a different concept from the networking configuration. We must represent them differently.

The name should stay in the TunnelConfig, not in the proxyString. The proxy string should be for configuring the VPN only, and the name is not needed for networking.

Reply:
'name' was not set before as far as I see. It seemed redundant and removed.